src/styles/variables.js

```diff
@@ -0,0 +1,24 @@
+const variables = {
+  black: "#000000",
+  white: "#ffffff",
+  brandBlue: "#0099cc",
+  brandDarkBlue: "#005f7f",
+  brandOrange: "#ff9933",
+  brandDarkOrange: "#ff6a33",
+  lightGrey: "#e5e5e5",
+  greyBlue: "#c1c9d1",
+  brandDark: "#324353",
+  faintBlue: "#f5f8fb",
+  faintGreen: "#f5fbf7",
+
+  headerUiBg: "#f3f8fc",
+  lowTranslucencyWhite: "rgba(255, 255, 255, 0.7)",
+
+  successColor: "#28a745",
+  warningColor: "#ffbf00",
+  warningBackground: "#fff9e8",
+  invalidColor: "#e46767",
+  invalidBackground: "#fbebe7"
+};
+
+export default variables;
```

Comment:
I recommend personnaly to always use named export. Random article to explain why => https://blog.neufund.org/why-we-have-banned-default-exports-and-you-should-do-the-same-d51fdc2cf2ad.

Here in that case I would either export a variable called color :)

Reply:
no changes?